Makefile

```diff
@@ -278,7 +278,12 @@ cross-image-%:
 
 .PHONY: test
 test: ## Run the unit test suite
-	${MAYBE_ENVIRONMENT_EXEC} cargo test --workspace --no-fail-fast --no-default-features --features ${DEFAULT_FEATURES} ${SCOPE} -- --nocapture
+	${MAYBE_ENVIRONMENT_EXEC} cargo test --quiet --workspace --no-fail-fast --no-default-features --features ${DEFAULT_FEATURES} ${SCOPE}
+
+.PHONY: test-benches
+test-benches: ## Run benchmarks in debug mode to trigger debug_assertions
+test-benches: export DEFAULT_FEATURES:="benches metrics-benches remap-benches"
```

Comment:
Is this the right syntax? I'm reading this as "target `test-benches` depends on files named `export` and `DEFAULT_FEATURES:="benches metrics-benches remap-benches"`"

Reply:
That is a good question. I tried to crib the syntax used in `test-components` but am not an expert in GNU make. My understanding is that this is an example of an "export directive".

Reply:
Indeed, I was also unfamiliar with this syntax, but it appears to be an example of [Target specific variable values](https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html#Target_002dspecific).

Reply:
Ah, gotcha. I had gone back to the GNU make documentation, and this didn't show up in the prerequisites sections. I see it now. Handy.